src/query/ts/metadata.go

```diff
@@ -47,16 +76,15 @@ const (
 
 // SeriesAttributes has attributes about the time series.
 type SeriesAttributes struct {
-	Type   MetricType
-	Source SourceType
+	M3Type            M3MetricType
+	PromType          PromMetricType
+	Source            SourceType
+	HandleValueResets bool
```

Comment:
Just following up here since GH removed the other comment about splitting this; was just thinking it's probably better not to have both prom and m3 types in a single `SeriesAttributes`, rather have them separate. Happy to leave it as is though 👍